code/web/src/modules/user/Profile.js

```diff
@@ -38,20 +38,22 @@ const Profile = (props) => (
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         {/*<Tile style={{}} image={icon}></Tile>*/}
-        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
+        <ImageTile style={{ borderRadius: '5px'}} object-fit='cover' width={300} height={300} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
```

Comment:
The size of the user image is now limited.

Reply:
Glad to see this worked in!